drivers/sensor/qdec_nxp_s32/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_QDEC_NXP_S32
```

Comment:
The ifdef is redundant here. It was already checked in `drivers/sensor/CMakeLists.txt`

Reply:
removed ifdef